fuzz/make_corpus.py

```diff
@@ -19,6 +19,7 @@
 import json
 import os.path
 
+
```

Comment:
What did we change that this was not linted before?

Reply:
yapf was complaining about a missing line (code style requires 2 empty lines at the end of the import block)